.github/workflows/test_latest_versions.yml

```diff
@@ -37,6 +45,4 @@ jobs:
           QISKIT_IBM_TOKEN: ${{ secrets.QISKIT_IBM_TOKEN }}
           QISKIT_IBM_INSTANCE: ${{ secrets.QISKIT_IBM_INSTANCE }}
         run: |
-          pver=${{ matrix.python-version }}
-          tox -e py${pver/./}
-          tox -e py${pver/./}-notebook
+          tox -e py,notebook,doctest --parallel --parallel-no-spinner
```

Comment:
This now runs the different tox jobs in parallel.
It also includes the `doctest` to be run for all versions, hence the `doctest.yml` workflow got deleted.